### PR TITLE
Release v0.15.4

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/blog/releases/v0.15.4.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Release v0.15.3"
```
Comment (nghialv) [inline docs/content/en/blog/releases/v0.15.4.md:2]: `v0.15.4` w

File: docs/content/en/blog/releases/v0.15.4.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+date: 2021-09-06
+description: >
+  Release v0.15.4
+---
+
+## Changelog since v0.15.3
+
+### Notable Changes
+* Mask secret and configuration values rendered as plan-preview result ([#2423](https://github.com/pipe-cd/pipe/pull/2423))
+
```
Comment (nakabonne) [inline docs/content/en/blog/releases/v0.15.4.md:13]: Nice

File: docs/content/en/blog/releases/v0.15.4.md (hunk 1 above)
Comment (nghialv) [inline docs/content/en/blog/releases/v0.15.4.md:2]:
```suggestion
title: "Release v0.15.4"
```

Comment (khanhtc1202) [reply docs/content/en/blog/releases/v0.15.4.md]: thanks 😄

**Pull request merged**